src/main/java/de/themorpheus/edu/taskservice/database/model/UserBanModel.java

```diff
@@ -0,0 +1,26 @@
+package de.themorpheus.edu.taskservice.database.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table
+public class UserBanModel {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private int bannedId;
+
```

Comment:
Add @ NotNull

Reply:
Do I need this if it's auto generated?

Reply:
the userId
and add column label

Reply:
NotNull and don't needed bc of nullable=false